mobile/mobile_menu.asm

```diff
@@ -456,12 +456,10 @@ String_4a34b:
 	next "もどる@"
 
 UnknownText_0x4a358:
-	; Delete the saved LOG-IN PASSWORD?
```

Comment:
These comments indicate the text content is known, so please don't leave the labels Unknown.